float32[] servo_pos
float32[] contact_sensors
float32[] imu_sensor